src/c++/tests/cc_client_test.cc

```diff
@@ -23,13 +23,15 @@
 // OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-#include "gtest/gtest.h"
 
+#include <fstream>
+
+#define TRITON_INFERENCE_SERVER_CLIENT_CLASS InferenceServerHttpClient
```

Comment:
Need to define this because of including `http_client.cc` below. Basically this macro needs to be defined before `common.h` gets included in order for a friend declaration to get compiled correctly. `common.h` gets included inside `grpc_client.h`, which is the first include in this file.